_maps/map_files/CTF/cruiser.dmm

```diff
@@ -540,10 +570,6 @@
 /turf/open/floor/pod/light,
 /area/centcom/ctf)
 "FP" = (
-/obj/structure/window/reinforced{
-	dir = 8;
-	resistance_flags = 64
```

Comment:
This was lost, not sure what it's for?

Reply:
thats indestructable iirc

Reply:
Thought so, I'll just replace it with the indestructible subtype then

Reply:
Oht here is no subtype lol I'll just add the var-edit back

Reply:
You can use `/turf/closed/indestructible/fakeglass` as a subtype